website/landing/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,71 @@
+import { Box, Text } from "../common";
+
+export const Footer: React.FC = () => {
+  return (
+    <Box
+      aria-labelledby="footer-label"
+      as="footer"
+      css={{
+        alignItems: "center",
+        display: "flex",
+        flexDirection: "column",
+        gap: "20px",
+        flexShrink: "O",
+        padding: "100px 0",
+      }}
+    >
+      <Text
+        css={{
+          fontFamily: "$base",
+          fontWeight: "$normal",
+          fontSize: "12px",
+          lineHeight: "15px",
+          textAlign: "center",
+          letterSpacing: "-0.0125em",
+        }}
+        id="footer-label"
+      >
+        powered by
+      </Text>
+      <Box
+        css={{
+          height: "32px",
+          width: "196px",
+        }}
+      >
+        <Box
+          as="svg"
+          css={{
+            color: "$darkTextPrimary",
+          }}
+          fill="none"
+          height="100%"
+          viewBox="0 0 197 32"
+          width="100%"
+        >
+          <title>CodeSandbox</title>
+          <path
+            clipRule="evenodd"
```

Comment:
I would split it into different files, maybe turn this into an asset (`.svg`), then we will leave `webpack` to take the decision of how to manage the cache of these files.